cebra/data/single_session.py

```diff
@@ -227,6 +227,13 @@ def _init_distribution(self):
                     self.dataset.continuous_index,
                     self.delta,
                     device=self.device)
+            # TODO(stes): Add this distribution from internal xCEBRA codebase at a later point
```

Comment:
why at a later point vs. now?

Reply:
It adds some third party code for sampling from the distribution and adds some complexity: Tests, integration in the sklearn API, etc.

It is not needed for the xcebra functionality.

I would vote to add this to the codebase in a separate PR, as it is orthogonal to xcebra (this distribution can also be used outside the context of attribution maps) and is not required to run any of the examples we would like to share.

Thoughts, @MMathisLab ?

Reply:
-> #244